library/lua/gui/widgets.lua

```diff
@@ -2877,13 +2870,6 @@ function FilteredList:onFilterChange(text)
     self:setFilter(text)
 end
 
-function FilteredList:onFilterChar(char, text)
-    if char == ' ' then
-        return string.match(text, '%S$')
```

Comment:
Ok, that makes sense. Thank you for the context. It does look like this change is probably the right thing to do for now, even if it was made without full understanding.

I did put in a mechanism for "don't accept this character *and* didn't pass it up the stack", though it may be too heavy handed for some situations. You can set `modal=true` on an EditField and it will eat all input, regardless of the result of on_char.